### DISCUSSION TITLE
Documentation Links

### DISCUSSION THREAD

**Discussion by jelling** [pmndrs/zustand#1733] [Ideas]
It would be helpful if there was link to [the full docs](https://docs.pmnd.rs/zustand/getting-started/introduction) on [the homepage](https://zustand-demo.pmnd.rs/). Or if the docs just became the homepage.

**Comment by dai-shi** [comment]
I'm fine with either way, but many people like the demo site.

cc: @dbritto-dev @chrisk-7777 @sewera

**Reply by chrisk-7777** [reply]
The home landing is pretty unique, I like it personally. 

How about a link to the docs in the primary nav here: 

![image](https://user-images.githubusercontent.com/55621012/230390423-f350aa3b-122a-4921-9167-c177ba335d12.png)

**Reply by dbritto-dev** [reply]
I like that, we can add a link to the docs. BTW I was wondering if we can have like two tabs for the code like javascript and typescript and show off the code for each one respectively. I would like to work on that.

**Reply by dai-shi** [reply]
Cool. You guys can send a PR and give a review. I can then merge and deploy.

**Reply by chrisk-7777** [reply]
PR added here: https://github.com/pmndrs/zustand/pull/1734

**Reply by dbritto-dev** [reply]
> PR added here: https://github.com/pmndrs/zustand/pull/1734

Approved. Let's merge it. cc @dai-shi
